src/about.html

```diff
@@ -8,6 +8,17 @@
   </head>
   <body>
     <h1>About AEG</h1>
+    <h2 style="font-family: Garamond, serif;">Introduction:</h2>
+    <p style="font-family: Garamond, serif;">
+    Alpha Existential Graphs evolved around 1896 by Charles Sanders Peirce which is a diagrammatic system of logic. In other words, this type of diagrammatic system has special symbols that are implemented in the graph and with the addition of ovals to present the negation, the graphs become more general to display the first-order logic with equality. 
+    </p>
+    <h2 style="font-family: Garamond, serif;">AEG:</h2>
+    <p style="font-family: Garamond, serif;">
+    Peirce was innovative as to how the structure of the AEG should look like, so he utilized alpha and beta symbols. Alpha and beta symbols were used to correspond to the logic of statements, predicates, and quantifiers.  These types of visual notation will express the logical expression that is depicted by Peirce. Another aspect of AEGs would be how they can be read in multiple ways that eliminates the need for additional special symbols. 
```

Comment:
> alpha and beta symbols.

I have no idea what "alpha and beta symbols." are, presumably you're talking about the alpha, beta, and gamma systems of existential graphs. Regardless this is wrong.